ontologies/_index.nq

```diff
@@ -67,6 +74,11 @@
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/2000/01/rdf-schema#isDefinedBy> <http://www.w3.org/ns/duv#> .
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/ns/rdfa#prefix> "duv" .
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/ns/rdfa#uri> <http://www.w3.org/ns/duv#> .
+<https://prefix.zazuko.com/earl:> <http://dbpedia.org/ontology/filename> "ontologies/earl.nq" .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/rdfa#PrefixMapping> .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/2000/01/rdf-schema#isDefinedBy> <https://www.w3.org/ns/earl#> .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/ns/rdfa#prefix> "earl" .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/ns/rdfa#uri> <https://www.w3.org/ns/earl#> .
```

Comment:
RDF/XML fails to parse `owl:Ontology` and I think that's why there is no `dcterms:description`. Add manually?

Reply:
I think it's getting parsed, here's what I get for description and label:

```
<https://www.w3.org/ns/earl> <http://www.w3.org/2000/01/rdf-schema#comment> "Formal schema of the Evaluation and Report Language (EARL) 1.0"@en <https://www.w3.org/ns/earl#> .
<https://www.w3.org/ns/earl> <http://www.w3.org/2000/01/rdf-schema#label> "Evaluation and Report Language (EARL) 1.0 Schema"@en <https://www.w3.org/ns/earl#> .
```

the reason it's not considered for the index is that the subject is `<https://www.w3.org/ns/earl>` when the vocab base IRI is `<https://www.w3.org/ns/earl#>`. I'll try a simple workaround, trimming trailing `#` and `/` for title and description